app/components/markdown/markdown.html.erb

```diff
@@ -1,3 +1,3 @@
-<div class="<%= class_attr %> rich-content">
+<div <%= attrs.merge(class: 'rich-content') %>>
```

Comment:
didn't end up using this component, in the end, but we added the AttributeBag quite a while ago and are refactoring as we touch the code. I originally used this, but later favored a suggestion by @tillprochaska and I don't think it makes sense to remove the change we want to introduce anyways.